usecases/modulecomponents/batch/batch.go

```diff
@@ -550,6 +550,71 @@ func VectorizeBatch[T []float32](ctx context.Context, objs []*models.Object, ski
 	return vecs, nil, errs
 }
 
+type batchObjectsVectorizer[T dto.Embedding] func(context.Context, []*models.Object, moduletools.ClassConfig) ([]T, models.AdditionalProperties, error)
```

Comment:
as this is not connected to the other batching implementation, could you move it to another file?

Reply:
I will move the naive batch and old batch logic to a separate file